test/utils/SafeTransferLib.t.sol

```diff
@@ -53,6 +53,8 @@ contract SafeTransferLibTest is Test {
     error TransferFailed();
     error ApproveFailed();
 
+    address constant CONSOLE = 0x000000000000000000636F6e736F6c652e6c6f67;
```

Comment:
You should not need to declare this `CONSOLE` constant, it should be accessible from `forge-std`.